descriptors/ModuleDescriptor-template.json

```diff
@@ -119,6 +143,10 @@
     {
       "id" : "authtoken",
       "version" : "2.0"
+    },
+    {
+      "id" : "authtoken2",
+      "version" : "1.0"
```

Comment:
Is this what I need to be doing here? Why would authtoken `optional`?

Reply:
that's right.